oshi-core/src/test/java/oshi/SystemInfoTest.java

```diff
@@ -131,7 +134,11 @@ public static void main(String[] args) {
         printSoundCards(hal.getSoundCards());
     }
 
-    private static void printComputerSystem(final ComputerSystem computerSystem) {
+    private static void printComputerVirtualization() {
+    	System.out.println("virtualization: " + Util.identifyVM());
+	}
+
+	private static void printComputerSystem(final ComputerSystem computerSystem) {
```

Comment:
Minor point, you've got tabs in your editor rather than spaces.  This causes unneeded whitespace changes in the diffs.   Can you set your editor to use 4 spaces rather than a tab?  If not, I can easily reformat and commit to your branch, but if it's an easy switch it'll make it easier.